ts/api/backend.ts

```diff
@@ -89,7 +89,7 @@ type CreateOrUpdateInstallationT = IPutApiRequestType<
   },
   "Authorization" | "Content-Type",
   never,
-  BasicResponseType<Installation>
+  BasicResponseType<NonEmptyString>
```

Comment:
does it really return a non empty string?

Reply:
Yes it returns `ok` in the body.